README.md

```diff
@@ -40,85 +57,97 @@ You would have to change a few things (see below, build from scratch)
-   - Select **Register** to create the application.
-1. On the app **Overview** page, find the **Application (client) ID** value and record it for later. You'll need it to configure the Visual Studio configuration file for this project.
-1. In the list of pages for the app, select **Authentication**.
-   -  1. In the **Suggested Redirect URIs for public clients (mobile, desktop)** list, check **https://login.microsoftonline.com/common/oauth2/nativeclient**.
-1. Select **Save**.
-1. In the list of pages for the app, select **API permissions**
+   - Under **Supported account types**, select **Accounts in any organizational directory and personal Microsoft accounts (e.g. Skype, Xbox, Outlook.com)**.
+1. Select **Register** to create the application.
+1. In the app's registration screen, find and note the **Application (client) ID**. You use this value in your app's configuration file(s) later in your code.
+1. In the app's registration screen, select **Authentication** in the menu.
+   - If you don't have a platform added, select **Add a platform** and select the **Public client (mobile & desktop)** option.
+   - In the **Redirect URIs** | **Suggested Redirect URIs for public clients (mobile, desktop)** section, select **https://login.microsoftonline.com/common/oauth2/nativeclient**
+
+1. Select **Save** to save your changes.
+1. In the app's registration screen, click on the **API permissions** blade in the left to open the page where we add access to the APIs that your application needs.
    - Click the **Add a permission** button and then,
-   - Ensure that the **Microsoft APIs** tab is selected
+   - Ensure that the **Microsoft APIs** tab is selected.
    - In the *Commonly used Microsoft APIs* section, click on **Microsoft Graph**
-   - In the **Delegated permissions** section, ensure that the right permissions are checked: **User.Read**. Use the search box if necessary.
-   - Select the **Add permissions** button
-
-
-### Step 3:  Configure the sample to use your Azure AD tenant
+   - In the **Delegated permissions** section, select **User.Read** in the list. Use the search box if necessary.
+   - Click on the **Add permissions** button at the bottom.
 
-In the steps below, "ClientID" is the same as "Application ID" or "AppId".
+##### Configure the  UWP App (UWP-App-calling-MSGraph) to use your app registration
 
-Open the solution in Visual Studio to configure the projects
+Open the project in your IDE (like Visual Studio) to configure the code.
+>In the steps below, "ClientID" is the same as "Application ID" or "AppId".
 
-#### Configure the sample to use your Azure AD app
+1. Open the `Native_UWP_V2\MainPage.xaml.cs` file
+1. Find the line ```private const string ClientId = "[Application Id pasted from the application registration portal]"``
+` and replace the existing value with the application ID (clientId) of the  `UWP-App-calling-MSGraph` application copied from the Azure portal.
 

```

Comment:
```suggestion
</details>
```

Reply:
Manual set does not contain expandable section as per current implementation.
Thanks!